src/config.py

```diff
@@ -27,6 +27,8 @@
 VOLUME_MULTIPLIER = 0.5
 # The maximum number of songs to download concurrently
 MAXIMUM_CONCURRENT_DOWNLOADS = 8
+# Link to Rick Astley's Never Gonna Give You Up
+YOUTUBE_URL = "https://www.youtube.com/watch?v=dQw4w9WgXcQ"
```

Comment:
Don't describe the value of the variable, describe the semantics of what it holds. also YOUTUBE_URL is probably not the best name. The comment and variable name should describe what this variable does, not what the string value is. e.g. the var might be better named PREVIEW_JUMP_URL and the comment saying something about it being the default jump url for the preview embed